.test/run.sh

```diff
@@ -605,6 +696,58 @@ fi
 	killoff
 	cid=$master_host
 	killoff
+
+	;&
+	validate_replica_user)
```

Comment:
a validation test should be like the first few tests in this file:
```
       validate_master_env)
 
       echo -e "Test: expect Failure  for MARIADB_REPLICATION_MODE='master' - none of MARIADB_REPLICATION_USER and MARIADB_REPLICATION_PASSWORD or MARIADB_REPLICATION_PASSWORD_HASH is specified\n"
 
       cname="mariadb-container-master-replica-fail-to-start-options-$RANDOM-$RANDOM"
       docker run  --rm  --name "$cname" \
               -e MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=1 \
               -e MARIADB_REPLICATION_MODE="master" \
               "$image" \
               && die "$cname should fail with incomplete options" 
        ;&
```

Important to notice, the script continues and doesn't die.

Reply:
Done 👍🏽